app/src/test/java/com/example/tasky/feature_agenda/presentation/AgendaViewModelTest.kt

```diff
@@ -0,0 +1,83 @@
+package com.example.tasky.feature_agenda.presentation
+
+import com.example.tasky.common.domain.TaskyState
+import com.example.tasky.feature_login.domain.repository.UserRemoteRepository
+import io.mockk.MockKAnnotations
+import io.mockk.coEvery
+import io.mockk.impl.annotations.RelaxedMockK
+import junit.framework.Assert.assertEquals
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.StandardTestDispatcher
+import kotlinx.coroutines.test.TestScope
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
+import org.junit.Before
+import org.junit.Test
+
+class AgendaViewModelTest {
+    @RelaxedMockK
+    private lateinit var taskyState: TaskyState
+
+    @RelaxedMockK
+    private lateinit var userRemoteRepository: UserRemoteRepository
+
+    private lateinit var viewModel: AgendaViewModel
+
+    private val testDispatcher = StandardTestDispatcher()
+    private val testScope = TestScope(testDispatcher)
+
+    @OptIn(ExperimentalCoroutinesApi::class)
+    @Before
+    fun setUp() {
+        MockKAnnotations.init(this, relaxUnitFun = true)
+        Dispatchers.setMain(testDispatcher)
+        viewModel =
+            AgendaViewModel(userRemoteRepository = userRemoteRepository, taskyState = taskyState)
+    }
+
+    @Test
+    fun formatInitials_given_firstName() {
+        // given
+        coEvery { taskyState.getName() } returns "firstName"
+
+        // when
+        runTest {
+            viewModel.getInitials()
+        }
+        val initials = viewModel.initials.value
+
+        // then
+        assertEquals("FI", initials)
```

Comment:
I would recommend to look into assertion libraries, since the default assertions aren't so readable for more complex ones. My personal favorite is AssertK. You can add it like this:
1. Version catalog declaration
```
[versions]
...
assertk = "0.28.1"

[libraries]
...
assertk = { module = "com.willowtreeapps.assertk:assertk", version.ref = "assertk" }
```
2. Reference it in your Gradle file:
```
dependencies {
    ...
    testImplementation(libs.assertk)
    androidTestImplementation(libs.assertk)
}
```
3. Then your assertions could look like this for example:
```
assertThat(initials).isEqualTo("PL")

assertThat(myList).hasSize(5)

assertThat(myList).contains(element)

assertThat(myBoolean).isTrue()
```
Reads like a book :D